static/js/containers/CollectionDetailPage_test.js

```diff
@@ -215,4 +220,29 @@ describe('CollectionDetailPage', () => {
     assert.deepEqual(state.collectionUi.editCollectionForm, makeInitializedForm(collection));
     assert.isTrue(state.commonUi.dialogVisibility[DIALOGS.COLLECTION_FORM]);
   });
+
+  it('showVideoMenu sets menu visibility to true', async () => {
+    let wrapper = await renderPage();
+    const state = await listenForActions([
+      SET_SELECTED_VIDEO_KEY,
+      SHOW_MENU,
+    ], () => {
+      wrapper.find("VideoCard").first().prop('showVideoMenu')();
+    });
+    const video = collection.videos[0];
+    assert.isTrue(state.commonUi.menuVisibility[video.key]);
+  });
+
+  it('closeVideoMenu sets menu visibility to false', async () => {
+    let wrapper = await renderPage();
+    const state = await listenForActions([
+      SET_SELECTED_VIDEO_KEY,
+      HIDE_MENU,
+    ], () => {
+      wrapper.find("VideoCard").first().prop('closeVideoMenu')();
+    });
+    const video = collection.videos[0];
+    assert.isFalse(state.commonUi.menuVisibility[video.key]);
+  });
```

Comment:
these tests can be pretty easily combined. something like...

```javascript
[
    ['showVideoMenu', true, [SET_SELECTED_VIDEO_KEY, SHOW_MENU]],
    ['closeVideoMenu', false, [SET_SELECTED_VIDEO_KEY, HIDE_MENU]]
  ].forEach(([action, expectedVisibility, expectedActions]) => {
    it(`${action} sets menu visibility to ${expectedVisibility}', async () => {
```

Reply:
OK